src/plugins/share/server/url_service/saved_objects/register_url_service_saved_object_type.ts

```diff
@@ -67,7 +68,7 @@ export const registerUrlServiceSavedObjectType = (
             },
           },
         },
-        // Information needed to load and execute a locator.
+       // Information needed to load and execute a locator.
         locatorJSON: {
```

Comment:
@vadimkibana I added `loccatorJSON` back.

Reply:
Yes, but maybe I'm wrong, maybe we can delete this field.

To clarify, if we delete this field, the data will still remain in the index, or will it be deleted? If the field mapping is removed, will we still be able to set this field and read data from it, or not?

Reply:
> if we delete this field, the data will still remain in the index, or will it be deleted

It will remain in the index (within `_source`).

> If the field mapping is removed, will we still be able to set this field and read data from it, or not?

The field won't be an attribute to your saved object anymore so setting and reading `locatorJSON` will mean refactoring to read from the raw es document and not from the saved object attributes.

Can you recall why `locatorJSON` was mapped with `index:false`?

Reply:
> The field won't be an attribute to your saved object anymore so setting and reading `locatorJSON` will mean refactoring to read from the raw es document and not from the saved object attributes.

Ah, so we cannot delete `locatorJSON` now, we need to make sure that data is read from the raw document instead of attributes first. So, we will need to do that in a follow up work. We are tracking this work here: https://github.com/elastic/kibana/issues/153484

I'll approve the `dynamic: false` bit.

> Can you recall why locatorJSON was mapped with index:false?

It is mapped with `index:false`, because we don't need to search/filter by that field.

Reply:
> [..] will mean refactoring to read from the raw es document and not from the saved object attributes.

@TinaHeiligers I'm wondering if that data will not be accessible from `.attributes`, where can we access that field from? I'm looking at a Saved object types and cannot figure it out: https://github.com/elastic/kibana/blob/main/packages/core/saved-objects/core-saved-objects-common/src/server_types.ts#L66-L104

Reply:
I've just checked, it looks like data is still available in `.attributes`.